HealthMonitoring.Persistence/MySqlDatabase.cs

```diff
@@ -40,10 +41,12 @@ private void CreateDatabaseIfNeeded()
                     CreateColumn(conn, "EndpointConfig", "Tags", "varchar(4096)");
                 if (!DoesColumnExists(conn, "EndpointConfig", "Password"))
                     CreateColumn(conn, "EndpointConfig", "Password", "varchar(64)");
+
                 if (!DoesColumnExists(conn, "EndpointConfig", "FirstTimeRegistered"))
-                    CreateColumn(conn, "EndpointConfig", "FirstTimeRegistered", "datetime");
+                    CreateColumnAndSetValue(conn, "EndpointConfig", "FirstTimeRegistered", "datetime not null", DateTime.UtcNow);
```

Comment:
If MySql going to allow you to create a not null column if table contains any records?